_includes/manuals/1.24/1.2_release_notes.md

```diff
@@ -8,7 +8,7 @@ Foreman can now provision hosts using Azure Resource Manager (AzureRM) as a Comp
 Foreman is now capable of performing Single Sign-On with the help of various OpenID Providers, such as Keycloak or Auth0, using the OIDC protocol. With the help of OpenID Providers, organizations can incorporate multiple authorization sources such as User Federations (LDAP or Kerberos), Social (such as GitHub, Twitter and others) and will be able to perform two-factor authentication (using OTP, smart card or code generation). The SSO feature enables an `External` user to login into Foreman using an OpenID Provider.
 
 #### Debian 10 (Buster) support
-Packages for Foreman and its various plug-ins are now being built for Debian 10 (Buster) as well.
+Packages for Foreman and its various plug-ins are now being built for Debian 10 (Buster) as well. With Buster support added in this release, the next release will drop support for Debian 9 (Stretch).
```

Comment:
Unsure if we should also add a note encouraging to upgrade. So either:

```suggestion
Packages for Foreman and its various plug-ins are now being built for Debian 10 (Buster) as well. The next Foreman release will drop support for Debian 9 (Stretch).
```

Or:

```suggestion
Packages for Foreman and its various plug-ins are now being built for Debian 10 (Buster) as well. The next Foreman release will drop support for Debian 9 (Stretch) and users are encouraged to upgrade.
```

Reply:
updated